src/dom/__tests__/errorSuppression.disabled.test.ts

```diff
@@ -5,10 +5,10 @@ describe('error output suppression (disabled) tests', () => {
 
   beforeAll(() => {
     process.env.RHTL_DISABLE_ERROR_FILTERING = 'true'
+    require('..')
   })
 
   test('should not patch console.error', () => {
-    require('..')
```

Comment:
This actually wasn't testing anything.  Requiring in the test wouldn't register the `beforeEach` and `afterEach` so it must be done before the test executes.  I discovered this because with the refactoring, this left a uncovered branch that is now covered.